config/initializers/importers.rb

```diff
@@ -40,6 +40,16 @@
   satisfaction.range = ENV.fetch('GOOGLE_SATISFACTION_RANGE', 'A:K')
 end
 
+Rails.configuration.x.google_sessions.tap do |sessions|
+  sessions.service_account_email = ENV['GOOGLE_SESSIONS_EMAIL']
+  sessions.key_data = Base64.decode64(ENV.fetch('GOOGLE_SESSIONS_KEY', ''))
+  sessions.key_secret = ENV['GOOGLE_SESSIONS_SECRET']
+  sessions.sheets = {
```

Comment:
Not planning to have multiple sheets at this stage. I did it this way to maximise the reuse of code to get data from google sheets. 

Not sure if the better to have one way to get the data from google or a less blotted config.